CMakeLists.txt

```diff
@@ -128,6 +148,9 @@ include_directories(
   SYSTEM ${CURL_INCLUDE_DIRS}
   include_directories(.) # to find src/fmt
 )
+if(SPDLOG_FMT_EXTERNAL)
+  include_directories(SYSTEM ${SPDLOG_INCLUDE_DIRS})
```

Comment:
I'm not sure about this line because before this line there is 
```include_directories(.) # to find src/fmt```. Looks like two different version of fmt will be included.

Reply:
Good point. I'll check that.

Reply:
So it looks like that line is unnecessary. fmt itself used to be bundled in `src/fmt`, but this was removed in adf036b. I think this line is effectively a no-op now, but in the interests of keeping my pull request focused on a single logical change, I didn't remove it.